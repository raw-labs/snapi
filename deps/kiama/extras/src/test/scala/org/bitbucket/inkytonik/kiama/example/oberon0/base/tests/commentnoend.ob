MODULE commentnoend;
 (* started off well,
   but didn't finish
END commentnoend.
