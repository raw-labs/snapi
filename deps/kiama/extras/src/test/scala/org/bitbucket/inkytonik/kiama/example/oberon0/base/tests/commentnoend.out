* errors
src/test/scala/org/bitbucket/inkytonik/kiama/example/oberon0/base/tests/commentnoend.ob:5:1:error: '*)' expected but end of source found

^
